pipelines/optimus/Optimus.wdl

```diff
@@ -202,7 +213,8 @@ workflow Optimus {
     call Count.CreateSparseCountMatrix {
       input:
         bam_input = PreCountSort.bam_output,
-        gtf_file = ModifyGtf.modified_gtf
+        gtf_file = ModifyGtf.modified_gtf,
+	  counting_mode = counting_mode
```

Comment:
This option does not matter, as CreateSparseCountMatrix is indifferent in the  function.